internal/domain/review_suggestion.go

```diff
@@ -0,0 +1,27 @@
+package domain
+
+import (
+	"github.com/invopop/jsonschema"
+)
+
+type ReviewSuggestion struct {
+	Comment  string `json:"comment"`
+	Position int    `json:"position"`
+}
+
+type ReviewSuggestionsResponse struct {
+	ReviewSuggestions []ReviewSuggestion `json:"review_suggestions"`
```

Comment:
The `GenerateSchema` function uses the empty interface return type, which is less clear than returning the specific schema type. If this function is intended to return a structured schema, consider changing the return type to `jsonschema.Schema` (or the appropriate type from the `jsonschema` package) for improved type safety and clarity.